src/sage/modular/drinfeld_modform/tutorial.py

```diff
@@ -0,0 +1,228 @@
+r"""
+Introduction to Drinfeld modular forms
+======================================
+
+This tutorial outlines the definitions, the notations, and the
+implementation of Drinfeld modular forms in SageMath. We assume that the
+reader has basic knowledge of classical modular forms, as we
+will often make analogies to this setting. We also assume little
+knowledge of Drinfeld modules. The interested reader can consult the
+Sage reference manual about this topic :ref:`Drinfeld modules`.
```

Comment:
Modify last sentence + reflow

```suggestion
This tutorial outlines the definitions, the notations, and the
implementation of Drinfeld modular forms in SageMath. We assume that the
reader has basic knowledge of classical modular forms, as we will often
make analogies to this setting. We also assume little knowledge of
Drinfeld modules; for this topic, the interested reader can consult the
SageMath reference manual: :ref:`Drinfeld modules`.
```